features/step_definitions/viewing_browse_steps.rb

```diff
@@ -52,8 +52,8 @@
 end
 
 Then(/^I see the list of top level browse pages alphabetically ordered$/) do
-  expect(page).to have_selector("div#root li:nth-child(1)", text: "Benefits"), "Benefits should appear first"
-  expect(page).to have_selector("div#root li:nth-child(2)", text: "Crime and justice"), "Crime and justice should appear second"
+  expect(page).to have_selector("ul.gem-c-cards__list li:nth-child(1)", text: /Benefits/), "Benefits should appear first"
```

Comment:
Using the `gem-c-` class in a test can be a bit of a hidden footgun - if the component gets renamed, the tests in the component gem will pass, the gem will be released, and then this test in collections will fail. 

As this is not testing for the use of the card component, could another non-gem selector be used here?

Reply:
That's a good point. We no longer have an attribute to hook onto that works without reference to something in the gem. The component currently doesn't allow you to set any custom classes or id's so one option is making that possible although our use case feels quite specific to this context.

Alternatively, I'm wondering if we actually need this test anymore? It seems to have been useful for when the component was custom to collections but now it's just literally testing a component that has its own tests in the gem.

Reply:
Agreed this test isn't helpful any more. I think that if you give the component a list of items I'd expect the order of the list to be respected - and to be tested at a component level if needed.

Reply:
I've decoupled the tests from the gem classes and they now trust the gem component to handle the alphabetical ordering.  We're now testing for the presence of links instead, similar to what we do [here](https://github.com/alphagov/collections/blob/c1e5ed91084bc354876c5ac9de29b7a02fb78ee9/features/step_definitions/latest_changes_steps.rb#L17). Although that example actually references the gem classes, I guess we have some inconsistencies in these tests that should be tidied up.

Reply:
Yep, there are definitely inconsistencies. It's good to aim for is not introducing _more_ problems, even if the old problems haven't been removed yet.